src/components/EditSections/EditExtendedInfo/EditExtendedInfo.js

```diff
@@ -100,7 +107,7 @@ class EditExtendedInfo extends Component {
             md={3}
           >
             <KeyValue label={<FormattedMessage id="ui-users.extended.folioNumber" />}>
-              {userId || '-'}
+              {isEditForm || '-'}
```

Comment:
Does this work as expected?

Reply:
I guess user id should be here  instead of the boolean